wwwroot/inc/database.php

```diff
@@ -4328,10 +4331,13 @@ function commitUpdateTag ($tag_id, $tag_name, $parent_id, $is_assignable)
 			(
 				'tag' => $tag_name,
 				'parent_id' => nullIfZero ($parent_id),
-				'is_assignable' => $is_assignable
+				'is_assignable' => $is_assignable,
+				'color' => nullIfEmptyStr($color)
 			),
 			array ('id' => $tag_id)
 		);
+		// remove all rack thumbnails
+                usePreparedDeleteBlade ('RackThumbnail', array ('1' => '1'));
```

Comment:
As a side note, a future improvement could be to have a function that would return a list of rack IDs that contain any of the objects from the given list of object IDs (which is trivial to get by the tag name). But this is not critical to get it running now.